### ISSUE TITLE
1.16.1 - Fabric server - Crash upon heads drop.

### ISSUE THREAD

**Issue by BuzzBazz** [sp614x/optifine#4278]
Upon killing parrots (not tested with other mobs) but as heads drop, my client boots me with crash reports.
Two reports should be added for further study. Problems only occur on clients with optifine. Others without have not had this problem happen to them. (unless it's my own doing, but I'm pretty sure it's only optifine as I'm the only one that is running it atm.)
[crash-2020-07-06_22.55.27-client.txt](https://github.com/sp614x/optifine/files/4881244/crash-2020-07-06_22.55.27-client.txt)
[crash-2020-07-06_22.58.51-client.txt](https://github.com/sp614x/optifine/files/4881245/crash-2020-07-06_22.58.51-client.txt)

**Comment by salfishies** [comment]
hey there im having the same problems as you are however i have tried other mobs and it seems as thought it only crashes when i kill parrots havent tried all of the mobs but so far no crashes until the parrot that is. This is ver odd

**Comment by BuzzBazz** [comment]
Indeed. Had some problems with 1.15.2 where placed playerheads would cause a stack error and crash the game, but only after we tried to add our own player heads to the list for the wandering trader datapack from vanilla tweaks.

**Comment by sp614x** [comment]
Duplicate of #4214
